### DISCUSSION TITLE
Please help me deploy my html css js project

### DISCUSSION THREAD

**Discussion by FireBird1998** [community/community#62168] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

https://github.com/FireBird1998/Weather-app-v1 
this is my repository URL Please have a look 

I have a plain js vanilla web however when I deployed with GitHub pages, the main js was missing from the source I have looked for a solution everywhere and still unable to see what is causing this problem. Please guys take a look  and Please help me

**Comment by radar-blip** [comment]
Have you tried changing this:     
`<script type="module" src="/main.js"></script>`

to this:
`<script type="module" src="./main.js"></script>`

(just add `.` in front of slash)

**Reply by GhostOf0days** [reply, marked as answer]
? The JavaScript file is at the root of the repo. The index file is also at the root of the repo. This shouldn't make a difference. JavaScript pathing works as if it is served from the page you're on (so if on home, relative pathing is from index.html) not from the JavaScript file. This therefore shouldn't fix it. I suggest @FireBird1998 to try moving the script from the head element to the body element.

**Reply by radar-blip** [reply]
That is an excellent suggestion. I think you may be right! 👍🏻

**Comment by FireBird1998** [comment]
Guys thank you for the awesome help, yes your suspicion was correct, as by adding / in front of the main.js in the index.html file, I was messing up the file path in GitHub server. I think it happened because of the platform difference between window and linux, 
may be in linux the / take you back one level up and by doing that repeatedly in my files I messed up all the paths and a slue of error happened. 

But Thank you guys for being so responsive, Its a pleasure, Thank you, 
Also all the error are now solved

**Reply by GhostOf0days** [reply]
Np. Also, code runs the same across platforms.
